test/terra/extensions/test_snapshot_density_matrix.py

```diff
@@ -22,6 +23,10 @@
 class TestSnapshotDensityMatrixExtension(QiskitAerTestCase):
     """SnapshotDensityMatrix extension tests"""
 
+    def setUpClass(cls):
+        super().setUpClass()
+        filterwarnings("default", category=DeprecationWarning)
```

Comment:
Do we want to limit the `default` category to only deprecation warnings emitted by a certain class?  I think we can set the `module` kwarg to the module the class is contained in, so we don't miss other deprecated functionality.

Reply:
I briefly tried to do this but couldn't figure out correct way to do it. Can you post code?

Reply:
For posterity: this was added in b79390e.